radis/lbl/base.py

```diff
@@ -3044,7 +3026,10 @@ def _check_inputs(self, mole_fraction, Tmax):
         # ... that wstep and broadening_max_width were not inadvertanly changed
         # ... (would have no effect as the waverange is calculated on SpectrumFactory
         # ... initialization)
-        assert self._wstep == self.params.wstep
+
+        # Checks there if there is change in wstep value if initial wstep != "auto"
+        if self.wstep != "auto":
```

Comment:
@erwanp I believe you wanted to say those [lines](https://github.com/radis/radis/pull/271#discussion_r644231439) for this part? If I remove this line then if we compute 2 Spectrum from a single SF object the `self.params.wstep` value will never be equal to `auto` and an Assertion Error will always be thrown `assert self.wstep == self.params.wstep`. If someone manually changes the wstep value then it is fine but if we generate a new spectrum from same SF object it will be wrong. Any possible solution to this? One way I can think is resetting the `self.params.wstep` value after calculating spectrum.